[2025-04-22T22:45:59.358+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:45:43.477683+00:00 [queued]>
[2025-04-22T22:45:59.364+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:45:43.477683+00:00 [queued]>
[2025-04-22T22:45:59.365+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-22T22:45:59.374+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): meteo_operations.save_meteo_data> on 2025-04-22 22:45:43.477683+00:00
[2025-04-22T22:45:59.379+0000] {standard_task_runner.py:57} INFO - Started process 3175 to run task
[2025-04-22T22:45:59.381+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.save_meteo_data', 'manual__2025-04-22T22:45:43.477683+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp7khhqide']
[2025-04-22T22:45:59.382+0000] {standard_task_runner.py:85} INFO - Job 53: Subtask meteo_operations.save_meteo_data
[2025-04-22T22:45:59.412+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:45:43.477683+00:00 [running]> on host ba2ad926eb92
[2025-04-22T22:45:59.465+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.save_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T22:45:43.477683+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-22T22:45:43.477683+00:00'
[2025-04-22T22:45:59.481+0000] {python.py:194} INFO - Done. Returned value was: Successfully inserted measurement into database
[2025-04-22T22:45:59.495+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.save_meteo_data, execution_date=20250422T224543, start_date=20250422T224559, end_date=20250422T224559
[2025-04-22T22:45:59.514+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-22T22:45:59.529+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
